src/main/java/uk/gov/hmcts/reform/slc/services/steps/getpdf/PdfMerger.java

```diff
@@ -1,7 +1,7 @@
 package uk.gov.hmcts.reform.slc.services.steps.getpdf;
 
 import org.apache.pdfbox.multipdf.PDFMergerUtility;
-import uk.gov.hmcts.reform.slc.services.servicebus.exceptions.PdfMergeException;
+import uk.gov.hmcts.reform.slc.services.steps.getpdf.exceptions.PdfMergeException;
```

Comment:
Would renaming `steps.getpdf` to `pdf` make sense?

Reply:
👏 . i'd leave for refactor anyway. at this time we are in the position to produce the quickest merge possible. seeing other stuff in the source code which ought to be amended/removed

Reply:
just removing service bus references here